### PR TITLE
Feature/metadata multiple sources

### PR DESCRIPTION
**Pull request by andresgnlez**
TO-DO: render list of sources in the main form.

### CODE REVIEW

File: components/admin/metadata/form/SourcesContentModal.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+// components
+import ContentGroup from 'components/ui/ContentGroup';
+import Source from 'components/admin/metadata/form/Source';
+import { SOURCE_ELEMENTS } from 'components/admin/metadata/form/constants';
+
+class SourcesContentModal extends React.Component {
+
+  componentWillMount() {
+    const { sources, setTmpSources } = this.props;
```
Comment (mbarrenechea) [inline components/admin/metadata/form/SourcesContentModal.js]: Try a dataset that doesn't have any source added. Could you please set the initial state as an array with one empty object? Just the same you are doing when you try to remove all sources...

image

Comment (mbarrenechea) [reply components/admin/metadata/form/SourcesContentModal.js]: Also, could you list the selected sources in the metadata form? At least string separated by commas

Comment (andresgnlez) [reply components/admin/metadata/form/SourcesContentModal.js]: image

Added source list.

**Comment by andresgnlez** [comment]
I've removed input validation for the time being.. This is giving us more problems than solutions in some cases.

**Pull request merged**